### ISSUE TITLE
Cortana Removal

### ISSUE THREAD

**Issue by Illumye** [ntdevlabs/tiny11builder#7]
Currently, you must manually remove Cortana with Powershell :

```powershell
Get-AppxPackage Microsoft.549981C3F5F10 | Remove-AppxPackage
```

**Comment by ntdevlabs** [comment]
There is already a similar line in the script, but it still doesn't work

**Comment by ianis58** [comment]
I think we could create a set of startup planned tasks, that would at each startup try to uninstall Cortana, Edge and Teams.
What do you think @ntdevlabs, @Illumye ?

**Comment by Illumye** [comment]
> I think we could create a set of startup planned tasks, that would at each startup try to uninstall Cortana, Edge and Teams.
> What do you think @ntdevlabs, @Illumye ?

As far I know, Edge is already uninstalled but there are some remnants like taskbar and desktop shortcut.

**Comment by ianis58** [comment]
Yes you're right, but it sometimes is installed back after some days / weeks. It comes from windows update.

**Comment by mgrak** [comment]
It would be nice to have all the junk uninstalled that windows updates throws back on randomly through a scheduled job/task.

**Comment by s3rj1k** [comment]
Why would you want to uninstall Edge?
It is so deeply integrated into system, uninstalling it would obviously break something, moreover uninstalling Edge and install what? Google Chrome?

**Comment by ianis58** [comment]
@s3rj1k in fact I run windows 11 with every trace of edge wiped (sometimes it's back after an update and I delete it again), but there is no issue at all doing this. I was thinking too that it could cause issues (with webviews or things like that) but I've seen no issue for 2 months now.
I prefer to remove it because it's not my default browser.

**Comment by s3rj1k** [comment]
@ianis58 I think be removing Edge you could possible break corporate HTA based portals. Still seems sub-optimal to remove it again and again after each update.

**Comment by ianis58** [comment]
@s3rj1k I understand your point an I agree with it, I think it depends on the usage/context.
It could be an option that can be toggled, but that would require some configuration for the tiny11 build.
In fact, I forked and heavily modified this tiny11builder repository (it's public in my repos), to make it more robust, and it can be configured to keep/remove more or less applications/packages.
